app/knights/create_knight.py

```diff
@@ -0,0 +1,26 @@
+class CreateKnight():

```

Comment:
Do not use verbs in class names and file names. Rename this class to `Knight`, file to `knight.py`, and module to `characters`.